### PR TITLE
[wip] new year, new design (system)

### PR DESCRIPTION
**Pull request by tlwr**
TLDR
----

Redesigns fourth wall to use the gov.uk design system and "cards".

The design language of the colours was unclear and it was unreadable on large
screens.

Purpose
-------

The purpose of this redesign is to:

- Make the design/colours less loud; when put on a build monitor next to an
alerting/monitoring dashboard a more muted design will not distract from the
more important monitor.

- Better information density; when put on a display in either a thin or very
wide configuration it was difficult to read the pull requests.

Not in scope
------------

- Actually fixing the colours - someone who can see them properly should have a
go at this

- Fixing the structure of this project. I've been meaning to move to using
Vue.js 2 or something similar so we can do template rendering in a better way,
however that should come in a separate PR.

- Fixing the tests, phantomjs problems on travis have haunted this repo for a while, when restructuring this repo / using a different JS framework, we can test this project better.

Screenshots
-----------

- todo

### CODE REVIEW

**Comment by tlwr** [comment]
Backed this out as pushed to the wrong upstream...
